charts/orchestrator/README.md

```diff
@@ -12,13 +12,12 @@ The following table lists the configurable parameters of the Orchestrator chart
 
 | Parameter                | Description             | Default        |
 | ------------------------ | ----------------------- | -------------- |
-| `includeCustomResources` | set to true to have the custom resources (KnativeEventing, KnativeServing, SonataFlow and SonataFlowPlatform). | `false` |
 | `sonataFlowOperator.enabled` | whether the operator should be deployed by the chart | `true` |
 | `sonataFlowOperator.subscription.namespace` | namespace where the operator should be deployed | `"openshift-operators"` |
 | `sonataFlowOperator.subscription.channel` | channel of an operator package to subscribe to | `"alpha"` |
 | `sonataFlowOperator.subscription.installPlanApproval` | whether the update should be installed automatically | `"Automatic"` |
 | `sonataFlowOperator.subscription.pkgName` | name of the operator package | `"sonataflow-operator"` |
-| `sonataFlowOperator.subscription.sourceImage` | catalog image of the development build. Unset it for the release build. | `"quay.io/jianrzha/kogito-serverless-operator-catalog:v2.0.0"` |
+| `sonataFlowOperator.subscription.sourceImage` | catalog image of the development build. Unset it for the release build. | `"quay.io/masayag/kogito-serverless-operator-catalog:v2.0.0-snapshot"` |
```

Comment:
Why not to use https://quay.io/organization/orchestrator ?

Reply:
We don't have any pipeline to build this and with time, we'll have to use a stable version of that from https://quay.io/organization/kiegroup but there is no release for it ATM. 
Or better, release from the openshift marketplace / operatorhub. 
I'll take this with Ricardo.

Reply:
thanks